src/DynamoCore/Logging/Analytics.cs

```diff
@@ -19,6 +21,15 @@ internal static void Start(DynamoModel model)
         {
             client = new DynamoAnalyticsClient();
             client.Start(model);
+            model.WorkspaceAdded += OnWorkspaceAdded;
+        }
+
+        static void OnWorkspaceAdded(WorkspaceModel obj)
+        {
+            if (obj is CustomNodeWorkspaceModel)
+                TrackScreenView("CustomWorkspace");
+            else
+                TrackScreenView("Workspace");
```

Comment:
How does tracking a `Workspace` screen (which would happen during Dynamo startup and when opening a DYN file) differ from tracking a `Geometry`/`Nodes` screen?

What do you think about tracking a `Geometry`/`Nodes` screen instead at this point, so that overall we have:

- Four different screens in the main window: `StartPage`, `Geometry`, `Nodes`, `CustomWorkspace`
- Four different pop-up window screens (so far): `Gallery`, `PackageManager`, `Python`, `Publish`